index.ts

```diff
@@ -63,8 +63,8 @@ export const s3routes = {
 
 export const s3middleware = {
     userMetadata: require('./lib/s3middleware/userMetadata'),
-    convertToXml: require('./lib/s3middleware/convertToXml'),
-    escapeForXml: require('./lib/s3middleware/escapeForXml'),
+    convertToXml: require('./lib/s3middleware/convertToXml').default,
```

Comment:
The real fix for these is switching to `import/export` in `index.ts`, similar to how `policies` are exported.

Reply:
Done in https://github.com/scality/Arsenal/pull/1882/commits/9e0cee849cba2a7607d7ca41ecc0dfaae51b7d38